src/templates/drizzle/nav.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="drizzle-Layout-nav drizzle-Nav" id="nav">
   <div class="drizzle-Nav-header">
-    <a href="/">
+    <a href="/" class="drizzle-u-flex drizzle-u-flexAlignItemsCenter">
```

Comment:
If there's any way these classes could be in the `logo.hbs` template instead, I think that would make it more straightforward for users to edit in the future.

Alternatively, you could make the `<span>` content a `<text>` element instead and treat the `<svg>` as one big image (which might eliminate the need for the flex stuff at all).